src/game/objects/pickup.c

```diff
@@ -33,6 +39,27 @@ int16_t g_PickUpBoundsUW[12] = {
     +45 * PHD_DEGREE,
 };
 
+static void PickUp_GetItem(
+    int16_t item_num, ITEM_INFO *item, ITEM_INFO *lara_item);
+
+static void PickUp_GetItem(
+    int16_t item_num, ITEM_INFO *item, ITEM_INFO *lara_item)
+{
+    if (lara_item->current_anim_state == AS_PICKUP) {
```

Comment:
Please swap the condition to reduce the indentation.

Reply:
I just realized I don't need this if check at all in here. I was originally going to do the check inside the function.